README.md

```diff
@@ -137,8 +142,67 @@ lg()
     fi
 }
 ```
+
 Then `source ~/.zshrc` and from now on when you call `lg` and exit you'll switch directories to whatever you were in inside lazyigt. To override this behaviour you can exit using `shift+Q` rather than just `q`.
 
+## Configuration
+
+All Lazygit configurations are stored on `<userConfigDir>/jesseduffield/lazygit/config.yml`, where
+`userConfigDir` is your OS-dependent user configurations folder, namely:
+
+- Windows: `%APPDATA%` (`C:\Users\<User>\AppData\Roaming`)
+- Linux/BSDs: `${XDG_CONFIG_HOME}` (`${HOME}/.config`)
+- MacOSX: `${HOME}/Library/Application Support`
+
+This file is auto-generated on first run.
+
+You can also get the default values with `lazygit -c`, which returns something like:
+
+```
+gui:
+  ## stuff relating to the UI
+  scrollHeight: 2
+  scrollPastBottom: true
+  mouseEvents: false # will default to true when the feature is complete
```

Comment:
this example config is a little outdated now. I would provide a link to `docs/Config.md` for the example

Reply:
Sorry I absolutely missed the docs - me and my bad habit of trying to write everything on the readme